iocBoot/iocCamera/st.cmd

```diff
@@ -1,9 +1,9 @@
 #!/opt/ad-aravis-epics-ioc/bin/linux-x86_64/Camera
 # -*- container-image: ghcr.io/cnpem/ad-aravis-epics-ioc
 
-cd /opt/ad-aravis-epics-ioc/iocBoot/iocCamera
+epicsEnvSet("IOC_CONFIG", "/opt/ad-aravis-epics-ioc/iocBoot/iocCamera")
 
-< envPaths
+< $(IOC_CONFIG)/envPaths
```

Comment:
In order to use the variables that the build system already defines for us, I think you could do something like:

```
< /full/path/to/envPaths
epicsEnvSet("IOC_CONFIG", "$(TOP)/iocBoot/$(IOC)")
```

It's unfortunate that we always need the full path to the binary and to `envPaths`...

Do you like this idea?

Reply:
> Do you like this idea?

I've though about that option. I seems to me equally redundant in terms of the base path. Automatically handling the IOC name might be good, but I don't think we'll often change that. I turned out sticking with the one I committed because I thought it would be clearer to have only a constant (env var) defining the base path for the entire script (but the shebang). The way you suggest, the path for `envPaths` seems to define it before we reach `IOC_CONFIG` declaration.

Anyway, I currently don't think the rationale I followed is the most important, though. I'm more concerned about how `envPath` is defined when we install the IOC in another directory. What value will `$(TOP)` have in that case? Will it be the build or installation directory? In addition, is `iocBoot` installable?

Reply:
> I've though about that option. I seems to me equally redundant in terms of the base path.

It kind of is. I think my reasoning for it is that another `st.cmd` for a different IOC can copy that line without having to change anything but the top bits (shebang and envPaths, which kinda have to be that way?). But I don't know if that actually checks out :sweat_smile: 

> The way you suggest, the path for envPaths seems to define it before we reach IOC_CONFIG declaration.

Indeed.

> What value will $(TOP) have in that case? Will it be the build or installation directory? In addition, is iocBoot installable?

I have no idea :thinking: But as we discussed, source code doesn't seem to take up that much storage, right?

Reply:
>  I think my reasoning for it is that another st.cmd for a different IOC can copy that line without having to change anything but the top bits.

I can quote your comment and add "(shebang and `IOC_CONFIG`, which kinda have to be that way?)" right after it, and it is still valid. Maybe it becomes a matter of which one is easier to identify where it should be changed. `envPath` seems less verbose in that sense, but maybe conceptually less close to the concept of path definition.

> But as we discussed, source code doesn't seem to take up that much storage, right?

Yeah, it doesn't. But if that's really really easy to make independent of the installation path, that would be nice to reduce yet another point of breakage for custom paths. But coming to think about that, both of these definitions (`envPath` and shebang) are hard to overcome. Unless we make the IOC binary installed somewhere `/usr/bin/env` can find, and envPath is a standard location. TBH, I can't see the downsides of it right away.